website/content/api-docs/json-jobs.mdx

```diff
@@ -1055,11 +1055,32 @@ README][ct].
   - `"noop"` - take no action (continue running the task)
   - `"restart"` - restart the task
   - `"signal"` - send a configurable signal to the task
+  - `"script"` - run a script
 
 - `ChangeSignal` - Specifies the signal to send to the task as a string like
   "SIGUSR1" or "SIGINT". This option is required if the `ChangeMode` is
   `signal`.
 
+- `ChangeScript` - Configures the script triggered on template change.  This
+  option is required if the `ChangeMode` is `script`.
+
+  The `ChangeScript` object supports the following attributes:
+
+  - `command` - Specifies the full path to a script or executable that is to be
+    executed on template change. Path is relative to the driver, e.g., if running
+    with a container driver the path must be existing in the container. This
+    option is required is the `change_mode` is `script`.
+  
+  - `args` - List of arguments that are passed to the script that is to be
+    executed on template change. 
+  
+  - `timeout` - Timeout for script execution specified using a label suffix like
+    "30s" or "1h". Default value is `"5s"`.
+  
+  - `fail_on_error` - If `true`, Nomad will kill the task if the script
+    execution fails. If `false`, script failure will be logged but the task will
+    continue uninterrupted. Default value is `false`.
```

Comment:
Oops...sorry I missed this before, but our JSON API uses the internal struct name format, which `PascalCase`.

```suggestion
  - `Command` - Specifies the full path to a script or executable that is to be
    executed on template change. Path is relative to the driver, e.g., if running
    with a container driver the path must be existing in the container. This
    option is required is the `change_mode` is `script`.
  
  - `Args` - List of arguments that are passed to the script that is to be
    executed on template change. 
  
  - `Timeout` - Timeout for script execution specified using a label suffix like
    "30s" or "1h". Default value is `"5s"`.
  
  - `FailOnError` - If `true`, Nomad will kill the task if the script
    execution fails. If `false`, script failure will be logged but the task will
    continue uninterrupted. Default value is `false`.
```